src/main/java/vazkii/pillar/StructureLoader.java

```diff
@@ -126,5 +127,41 @@ public static String getStructureNBTLocation(String jsonFileName) {
 	public static String jsonifySchema(StructureSchema schema) {
 		return gson.toJson(schema);
 	}
-	
+
+	public static StructureSchema getDefaultSchema() {
```

Comment:
The template is 1) not valid JSON (there's a trailing `,` that shouldn't be there on `rarity`) and 2) unread by StructureLoader since `generatorType` is blank. I was unsure how to handle it, so I opted for a getter. Originally I was going to use the template until I noticed those. Didn't know if I should change it or not.